vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -314,6 +313,7 @@
     </ClInclude>
     <ClInclude Include="LifecycleState.h" />
     <ClInclude Include="Modules\AppStateModule.h" />
+    <ClInclude Include="Modules\AsyncStorageModuleWin32.h" />
```

Comment:
>AsyncStorageModuleWin32.h [](start = 32, length = 25)

Having win32 specific module in a UWP project seems to be wrong.

Reply:
This file should probably be delted, but I didn't want to take that step in this change.
@htpiv mentioned the base React API (AsyncStorageModule) was dropped in 0.62.

Reply:
Removed.